src/style-flight-main.scss

```diff
@@ -5,9 +5,11 @@
 
 // Import Auro tokens
 @import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import '../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
```

Comment:
@aesthetiques Can we make the path to the node_modules the same as above?

Reply:
Yeah. Updated for consistency.